R/collect.R

```diff
@@ -152,7 +152,13 @@ collect_predictions.tune_results <- function(x, summarize = FALSE, parameters =
   } else {
     x <- collector(x, coll_col = coll_col)
   }
-  dplyr::relocate(x, dplyr::starts_with(".pred"))
+
+  x <- dplyr::relocate(x, dplyr::starts_with(".pred"))
```

Comment:
I played around with how to do this too. I think the order should be

 1. `.pred`
 2. `.pred_class`, `.pred_time` (and so on for other non class-probability estimates)
 3. `.pred_{class level}`
 
I just thought of this, which seems to work: 

``` r
library(tidymodels)
test <- tibble(.pred_a = 1:3, .pred_b = 1:3, .pred_class = 1:3, 
               .pred = 1:3, .pred_time = 1:3)
test %>% select(any_of(".pred"), any_of(".pred_class"), 
                any_of(".pred_time"), starts_with(".pred"))
#> # A tibble: 3 × 5
#>   .pred .pred_class .pred_time .pred_a .pred_b
#>   <int>       <int>      <int>   <int>   <int>
#> 1     1           1          1       1       1
#> 2     2           2          2       2       2
#> 3     3           3          3       3       3
```

<sup>Created on 2024-01-12 with [reprex v2.0.2](https://reprex.tidyverse.org)</sup>

Reply:
done in [`4376325` (#803)](https://github.com/tidymodels/tune/pull/803/commits/4376325dbeee428c10be81ef31adfc08ddf699af)